deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/RegisterManager.java

```diff
@@ -36,14 +36,6 @@
     @SerializedName(MODULE_ID_TAG)
     protected String moduleId = null;
 
-    /**
```

Comment:
Seems like this generated code might need to stay?

I've left the serialized fields alone in the past.

Reply:
I'd suppress the warning here, yeah. This is a valid field in the json payload

Reply:
Actually, this entire class seems to be unused outside of test code. You can probably just delete the class and its tests